sdk-workflows/src/main/java/io/dapr/workflows/runtime/WorkflowContext.java

```diff
@@ -66,6 +67,92 @@ public interface WorkflowContext {
    * @return Asynchronous task to {@code await()}.
    */
   Task waitForExternalEvent(String eventName, Duration timeout);
+  
+  
+
+  /**
+   * Asynchronously invokes an activity by name and with the specified input value and returns a new {@link Task}
+   * that completes when the activity completes. If the activity completes successfully, the returned {@code Task}'s
+   * value will be the activity's output. If the activity fails, the returned {@code Task} will complete exceptionally
+   * with a {@link TaskFailedException}.
+   *
+   * @param name       the name of the activity to call
+   * @param input      the serializable input to pass to the activity
+   * @param options    additional options that control the execution and processing of the activity
+   * @param returnType the expected class type of the activity output
+   * @param <V>        the expected type of the activity output
+   * @return a new {@link Task} that completes when the activity completes or fails
+   */
+  <V> Task<V> callActivity(String name, Object input, TaskOptions options, Class<V> returnType);
+
+  /**
+   * Asynchronously invokes an activity by name and returns a new {@link Task} that completes when the activity
+   * completes. See {@link #callActivity(String, Object, TaskOptions, Class)} for a complete description.
+   *
+   * @param name the name of the activity to call
+   * @return a new {@link Task} that completes when the activity completes or fails
+   * @see #callActivity(String, Object, TaskOptions, Class)
+   */
+  default Task<Void> callActivity(String name) {
+    return this.callActivity(name, Void.class);
+  }
```

Comment:
Am I missing something here?
shouldn't this be `this.callActivity(name, null, null, Void.class);` ?

Reply:
This method calls `default <V> Task<V> callActivity(String name, Class<V> returnType)` in line 123

Reply:
Do you think we should change that to `this.callActivity(name, null, null, Void.class);`?

Reply:
> Do you think we should change the method body here to `this.callActivity(name, null, null, Void.class);`?

This will be a heavily nitpick comment, so I leave it for you to decide :)

On lines 109-110, we have a similar approach where the full method is called instead of the templated version (lines 139-140).  For consistency, I would do it in the same fashion, whatever it is :)

Reply:
yeah, makes sense :)